bin/ingest.py

```diff
@@ -352,12 +358,18 @@ def main(args):
                 name = PersonName.from_element(node)
                 ids = people.get_ids(name)
                 if len(ids) > 1:
-                    print(
-                        f"WARNING ({anth_id}): ambiguous author {name}, defaulting to first of {ids}"
-                    )
+                    choice = -1
+                    while choice < 0 or choice >= len(ids):
+                        print(
+                            f"({anth_id}): ambiguous author {name}; Please choose from the following:"
+                        )
+                        for i, id_ in enumerate(ids):
+                            print(f"[{i}] {id_} ({people.get_comment(id_)})")
+                        choice = int(input("--> "))
+
                     ambiguous[anth_id] = (name, ids)
```

Comment:
Simple prompt to resolve ambiguities. Could be made more useful (e.g., by providing links to author pages, as well as opening the PDF).